README.md

```diff
@@ -165,6 +165,11 @@ echo "Hello SwiftNIO" | nc localhost 9999
 
 If all goes well, you'll see the message echoed back to you.
 
+To generate an xcodeproj to work on SwiftNIO from Xcode 
```

Comment:
Couple of nitpicks (because I know the NIO team are picky! 😜 ):
* the line ending should probably have an `:` at the end before the code block.
* `xcodeproj` should probably be changed to `Xcode project`
* `from` should be changed to `in`